.travis.yml

```diff
@@ -14,13 +14,20 @@ os:
   - osx
   - windows
 
-script: npx nyc -s npm run test:node -- --timeout 60000
+script:
+  - npm install ipfs
+  - npm install ipfs-http-client
+  - npm install go-ipfs-dep
```

Comment:
For discussion: we should install the versions this module depends on otherwise updating them may cause the tests for this module to break.

Reply:
Can do, though this module depends on `*` so it already does install the version this module depends on, for better or for worse :trollface: 

We could change it to be an actual version, but that would imply we require a certain version.

On that, looking through the code quickly, the API methods we call are:

`ipfs.id`
`ipfs.version`
`ipfs.init`
`ipfs.start`
`ipfs.stop`

I guess it's feasible that we could refactor those, but they are pretty low-churn.

The `.id` invocation looks a bit weird to me - it's used to set a `peerId` property on a node which is then used by the tests so we could probably remove it (in another PR).

We could probably remove `.version` too, as the implementation version is in the output of `node.api.id` so it seems redundant.